src/cargo/core/features.rs

```diff
@@ -757,6 +757,7 @@ unstable_cli_options!(
     build_std: Option<Vec<String>>  = ("Enable Cargo to compile the standard library itself as part of a crate graph compilation"),
     build_std_features: Option<Vec<String>>  = ("Configure features enabled for the standard library itself when building the standard library"),
     cargo_lints: bool = ("Enable the `[lints.cargo]` table"),
+    checksum_freshness: bool = ("Use a checksum to determine if output is fresh rather than filesystem mtime"),
```

Comment:
Unstable features should also be documented at https://doc.rust-lang.org/nightly/cargo/reference/unstable.html

See `src/doc/src/reference/unstable.md`

Reply:
Resolved in 6cf92e1f1677ebe030ddfabea8782e2f721378ce